### PR TITLE
Improve docs of gru

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
This is PR for improving docs of functions/links. Related issue: #2182

- StatelessGRU
- StatefulGRU

### CODE REVIEW

File: chainer/links/connection/gru.py [hunk 1]
```diff
@@ -77,6 +79,30 @@ class StatelessGRU(GRUBase):
 
     .. seealso:: :class:`~chainer.links.StatefulGRU`
 
+    .. admonition:: Example
+
+        There are several ways to make a StatefulGRU link.
```
Comment (niboshi) [inline chainer/links/connection/gru.py]: This should be StatelessGRU, I guess.
Also please put ``` ``  `` ``` to make it a code fragment.

File: chainer/links/connection/gru.py [hunk 2]
```diff
@@ -77,6 +79,30 @@ class StatelessGRU(GRUBase):
...
+        Let a two-dimensional input array :math:`x`:
```
Comment (niboshi) [inline chainer/links/connection/gru.py]: How about "Let x be a ..."? I think that would be more natural.

File: chainer/links/connection/gru.py [hunk 3]
```diff
@@ -137,6 +165,33 @@ class StatefulGRU(GRUBase):
         * :class:`~chainer.links.GRU`: an alias of
           :class:`~chainer.links.StatefulGRU`
 
+    .. admonition:: Example
+
+        There are several ways to make a StatefulGRU link.
+        Let a two-dimensional input array :math:`x`:
```
Comment (niboshi) [inline chainer/links/connection/gru.py]: Ditto. "Let x be a ..."

**Comment by keisuke-umezawa** [comment]
@niboshi Thank you for the review! I have fixed them.

**Comment by niboshi** [comment]
Thank you!
jenkins, test this please

**Comment by niboshi** [comment]
LGTM!

**Pull request merged**